.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: python
 python:
-  - "3.5"
+  - "3.6"
```

Comment:
Is changing these required for this PR? There are some `InterpreterNotFound` errors in there, which might be fixed by #262.